charts/smokey/templates/cronjobs/all-smoke-tests.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: all-smoke-tests
+spec:
+  schedule: "*/10 * * * *"
```

Comment:
Do the tests actually finish within 10 minutes? :) Might be worth giving it a bit more leeway.

Reply:
Caveated that I couldn't actually get "all smoke tests" running because of missing apps. Checking what we currently have running via Jenkins, it seems to take less than three minutes for smoke tests to complete, so 10 minutes might be alright. Happy to give it some more time though!